Leaving a channel crashes the app

I had a couple of empty public channels that I wanted to remove to make room.
Once I click on "Leave chat", the app crashes with the following critpci error message:
```
DBG 2020-12-29 09:30:55.108-05:00 callPrivateRPC                             topics="rpc" tid=15436 file=core.nim:20 rpc_method=wakuext_deleteMessagesByChatID
ERR 2020-12-29 09:30:55.108-05:00 TODO: if user is logged in, logout         topics="main" tid=15436 file=nim_status_client.nim:145
Traceback (most recent call last)
/home/jonathan/dev/nim-status-client/src/nim_status_client.nim(205) nim_status_client
/home/jonathan/dev/nim-status-client/src/nim_status_client.nim(147) mainProc
/home/jonathan/dev/nim-status-client/vendor/nimqml/src/nimqml/private/qqmlapplicationengine.nim(40) delete
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [Makefile:450: run-linux] Error 1
```

The channel has been left however. On reopen, I can no longer see it.

Experienced on Ubuntu 20
Latest master (6afe9088e90759a6663c23185460ff922838fbbf)

--- a/src/app/chat/view.nim
+++ b/src/app/chat/view.nim
@@ -397,8 +397,9 @@ QtObject:
 
   proc removeChat*(self: ChatsView, chatId: string) =
     discard self.chats.removeChatItemFromList(chatId)
-    self.messageList[chatId].delete
-    self.messageList.del(chatId)
+    if (self.messageList.hasKey(chatId)):
+      self.messageList[chatId].delete
+      self.messageList.del(chatId)
 
   proc clearChatHistory*(self: ChatsView, id: string) {.slot.} =
     self.status.chat.clearHistory(id)

--- a/ui/app/AppLayouts/Profile/Sections/MyProfileContainer.qml
+++ b/ui/app/AppLayouts/Profile/Sections/MyProfileContainer.qml
@@ -72,10 +72,11 @@ Item {
             id: profileName
             text: ensName !== "" ? ensName : username
             anchors.left: profileImgContainer.right
-            anchors.leftMargin: 8
+            anchors.leftMargin: Style.current.halfPadding
             anchors.top: profileImgContainer.top
-            font.weight: Font.Medium
-            font.pixelSize: 15
+            anchors.topMargin: 4
+            font.weight: Font.Bold
+            font.pixelSize: 20
         }
 
         Address {
@@ -94,7 +95,7 @@ Item {
             height: 32
             width: 32
             radius: 8
-            anchors.verticalCenter: parent.verticalCenter
+            anchors.verticalCenter: profileImgContainer.verticalCenter
             icon.name: "qr-code-icon"
             iconColor: Style.current.textColor
             onClicked: {
@@ -118,8 +119,9 @@ Item {
             asynchronous: true
             fillMode: Image.PreserveAspectFit
             source: profileModel.qrCode(pubkey)
+            anchors.verticalCenterOffset: 20
             anchors.horizontalCenter: parent.horizontalCenter
-            anchors.verticalCenter: parent.verticalCenter + 20
+            anchors.verticalCenter: parent.verticalCenter
             height: 312
             width: 312
             mipmap: true

--- a/ui/shared/status/StatusChatInput.qml
+++ b/ui/shared/status/StatusChatInput.qml
@@ -640,8 +640,7 @@ Rectangle {
                 font.pixelSize: 15
                 font.family: Style.current.fontRegular.name
                 wrapMode: TextArea.Wrap
-                anchors.bottom: messageInput.bottom
-                anchors.top: messageInput.top
+                height: parent.height
                 placeholderText: qsTr("Type a message")
                 placeholderTextColor: Style.current.secondaryText
                 selectByMouse: true

